### PR TITLE
runfix: android mobile browser for oauth

### PR DESCRIPTION
**Pull request by tlebon**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

Fixes an issue with android browser usage.

### Causes (Optional)

apparently there is chrome and chrome mobile for android. chrome mobile was missing

### CODE REVIEW

**Review by V-Gira — APPROVED** [review]

File: src/script/auth/module/action/RuntimeAction.ts [hunk 1]
```diff
@@ -31,11 +31,16 @@ export class RuntimeAction {
...
       const outlookBrowser = 'unknown';
+
       const isMobileSupportedBrowser = () => {
         return (
           Runtime.isMobileOS() &&
-          (Runtime.isSafari() || Runtime.isChrome() || Runtime.getBrowserName() === androidBrowser)
+          (Runtime.isSafari() ||
+            Runtime.isChrome() ||
+            Runtime.getBrowserName() === androidBrowser ||
+            Runtime.getBrowserName() === chromeMobile)
```
Comment (przemvs) [inline src/script/auth/module/action/RuntimeAction.ts]: [androidBrowser, chromeMobile].includes(Runtime.getBrowserName());

WDYT ?

File: src/script/auth/module/action/RuntimeAction.ts [hunk 2]
```diff
@@ -31,11 +31,16 @@ export class RuntimeAction {
   checkSupportedBrowser = (): ThunkAction<void> => {
     return (dispatch, getState, {getConfig}) => {
       const androidBrowser = 'android browser';
+      const chromeMobile = 'chrome mobile';
```
Comment (przemvs) [inline src/script/auth/module/action/RuntimeAction.ts]: Maybe we can move this variables outside the class ?

File: src/script/auth/module/action/RuntimeAction.ts (hunk 1 above)
Comment (tlebon) [reply src/script/auth/module/action/RuntimeAction.ts]: yea. in the future can you suggest this as a code change? makes it slightly easier to solve.

File: src/script/auth/module/action/RuntimeAction.ts (hunk 2 above)
Comment (tlebon) [reply src/script/auth/module/action/RuntimeAction.ts]: done

**Review by przemvs — APPROVED** [review]
GJ!

**Pull request merged**